homeassistant/components/tractive/const.py

```diff
@@ -6,7 +6,11 @@
 
 RECONNECT_INTERVAL = timedelta(seconds=10)
 
+ATTR_DAILY_GOAL = "daily_goal"
+ATTR_MINUTES_ACTIVE = "minutes_active"
+
 TRACKER_HARDWARE_STATUS_UPDATED = "tracker_hardware_status_updated"
 TRACKER_POSITION_UPDATED = "tracker_position_updated"
+TRACKER_ACTIVITY_STATUS_UPDATED = "tractive_tracker_activity_updated"
```

Comment:
The signals should be prefixed by the domain

Reply:
Will be fixed by: https://github.com/home-assistant/core/pull/54532

https://github.com/home-assistant/core/pull/54143#issuecomment-894207201

Reply:
Not for the new signal added in this PR